stream.c

```diff
@@ -23,11 +23,32 @@
 #include "socket.h"
 #include "stats.h"
 #include "thread.h"
+#ifdef WITH_TCPDIRECT
+#include "tcpdirect.h"
+#endif
 
 static void *stream_alloc(struct thread *t)
 {
         const struct options *opts = t->opts;
 
+#ifdef WITH_TCPDIRECT
+        if (!t->f_mbuf && t->opts->tcpd_gpu_pci_addr) {
+                if (tcpdirect_cuda_setup_alloc(t->opts, &t->f_mbuf, t)) {
+                        LOG_ERROR(t->cb, "%s: failed to setup tcpdirect CUDA socket",
+                                  __func__);
+                        exit(1);
+                }
+        }
+
+        if (!t->f_mbuf && t->opts->tcpd_nic_pci_addr) {
+                if (udmabuf_setup_alloc(t->opts, &t->f_mbuf)) {
```

Comment:
Weird to have lingering udmabuf code, but OK, lets please follow up. Can you file a bug to track the udmabuf support?

Reply:
filed a child bug